src/test/scala/rx/lang/scala/ObservableTest.scala

```diff
@@ -428,4 +428,15 @@ class ObservableTests extends JUnitSuite {
     assertEquals(expectedMap3, m3.toBlocking.single)
   }
 
+  @Ignore // Don't run it. Only test if it can be compiled
+  def testErrorSignature() {
+    val o: Observable[Int] = Observable.error(new RuntimeException("Oops")).map(v => v)
+    println(o.toBlocking.single)
+
+    val x = 1
+    println(x + (Observable.error(new RuntimeException("Oops")).toBlocking.single: Int))
```

Comment:
Some people might prefer `Observable.error[Int](...)` over a `... : Int` ascription... ;-)
But I guess such examples will be very rare, only occur in constructed examples, so I'd be ok with `def error(exception: Throwable): Observable[Nothing]` (but note that it's a breaking change).